database-plugins/src/main/java/co/cask/hydrator/plugin/db/batch/source/ETLDBInputFormat.java

```diff
@@ -39,17 +40,22 @@
  * Class that extends {@link DBInputFormat} to load the database driver class correctly.
  */
 public class ETLDBInputFormat extends DBInputFormat {
+  public static final String AUTO_COMMIT_ENABLED = "co.cask.hydrator.db.autocommit.enabled";
```

Comment:
nit: although this is purely internal, any reason for using this long, almost package-name like parameter? Can we just use ``autocommit.enabled``?